perf-event/Cargo.toml

```diff
@@ -22,7 +22,10 @@ default = ["hooks"]
 
 [dependencies]
 bitflags = "1.3"
+bytes = { git = "https://github.com/phantomical/bytes", branch = "buf-ne" }
```

Comment:
Just calling attention to this for now so that this doesn't get merged before a version of bytes with these changes is released :)

I'm still waiting for the PR to bytes to be merged but once that's happened I will let you know: https://github.com/tokio-rs/bytes/pull/576